ludwig/benchmarking/summary_dataclasses.py

```diff
@@ -1,17 +1,21 @@
 import os
 from dataclasses import dataclass
+from statistics import mean
 from typing import Any, Dict, List, Union
 
+from torch.autograd.profiler_util import _format_memory, _format_time
+
+import ludwig.modules.metric_modules  # noqa: F401
 from ludwig.globals import MODEL_HYPERPARAMETERS_FILE_NAME, REPORT_JSON
-from ludwig.modules.metric_registry import get_metric_classes
+from ludwig.modules.metric_registry import get_metric_classes, metric_feature_registry  # noqa: F401
 from ludwig.utils.data_utils import load_json
 
 
 @dataclass
-class ExperimentSummary:
+class MetricsSummary:
```

Comment:
I think I prefer `ExperimentSummary`, but curious to know why you wanted to rename?

Reply:
replaced it because now we have `metrics` (accuracy, r2, etc.) and `resource usage metrics`. Both metrics and resource usage metrics are part of an `experiment`. I changed it to prevent confusion. wdyt?